js/index.js

```diff
@@ -1,6 +1,100 @@
+        example above when we go through the 5th loop the name `Jose` has nothing after the letter `e` so doesn't matter what comes next.*/
+    } else if( (alphabet.indexOf(hacker1[index])) <  (alphabet.indexOf(hacker2[index]))  ){
+        console.log(`The driver's name goes first.`)
+        break
+    }   else if ( (alphabet.indexOf(hacker1[index])) >  (alphabet.indexOf(hacker2[index])) ) {
+        console.log(`Yo, the navigator goes first definitely.`)
+        break
+    }
+    
+}
+
+
+// bonus 1
+const paragraph = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer a ligula vitae dolor pretium tempus pharetra id nisi. Curabitur mattis lorem et tristique rutrum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Donec odio est, vulputate et ullamcorper non, blandit ut nunc. Sed cursus, nisi in eleifend congue, ipsum diam dapibus massa, a mollis nibh turpis at augue. Aliquam velit dui, semper a libero ut, sollicitudin imperdiet ipsum. Quisque blandit viverra lacus. Mauris et rutrum magna. Vestibulum quis lorem eu massa iaculis finibus. Proin vitae velit pulvinar, ornare augue vel, condimentum turpis. Aenean vestibulum aliquet arcu, at gravida ante cursus vitae. Nunc felis sapien, tincidunt eu pellentesque vel, lacinia non sem. Suspendisse cursus arcu sapien, sit amet venenatis lectus sodales in. Aliquam lorem felis, faucibus vitae scelerisque bibendum, commodo eget ipsum.
+
+Phasellus vitae ipsum non justo finibus consequat in ac felis. Donec at lectus porttitor, pharetra nisl eu, facilisis risus. Cras elit libero, euismod vehicula libero ac, sollicitudin lacinia ex. Nullam nec pellentesque magna. In ultrices nibh eu laoreet ultrices. In at aliquet diam. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque porttitor purus lacinia, dictum urna a, eleifend mi. Aenean fermentum tempus commodo. Proin ut aliquam ante. Fusce dapibus ipsum at eros egestas, at sollicitudin lectus finibus. Quisque placerat, lacus sed lobortis elementum, nibh urna pharetra mi, eget ornare augue neque mollis velit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus luctus, ligula in scelerisque commodo, tortor felis lacinia ex, et faucibus sapien tellus eu purus.
+
+Nullam quis lobortis ipsum. Proin ut semper elit. Suspendisse efficitur suscipit lacinia. Quisque egestas ullamcorper ante pretium ornare. Morbi quis interdum tellus. Donec vel tellus consequat, sollicitudin orci nec, tincidunt arcu. Cras ligula augue, iaculis sed odio vitae, ornare consequat justo. Etiam eget gravida felis. Ut facilisis ultrices dui eu ultrices. Quisque fermentum, tellus porta condimentum efficitur, libero nisi sodales erat, ut convallis ante augue nec turpis. Etiam ornare lacinia laoreet.`
+
+const paragraphArray = paragraph.split(' ')
+const isEt = word =>  {if(word==='et') return word}
```

Comment:
aí sim, ja criou uma `arrow function` 👏  se for colocar `{}` sempre idente seu código pra facilitar a leitura. Mas aqui se quisesse, se não me engano, nem precisaria dessas `{}` e aqui seria um bom lugar pra testar o ternário!